flashcards/backend/decks/tests/test_flashcard_model.py

```diff
@@ -0,0 +1,38 @@
+from datetime import date
+from django.contrib.contenttypes.models import ContentType
+
+
+def test_correct_gui_representation(flashcard, remove_test_data):
+    assert str(flashcard) == "test"
+
+
+def test_flashcard_creation(deck, remove_test_data, user, difficulty, flashcard, category, text):
+    assert flashcard.deck == deck.pk
+    assert flashcard.category == category.pk
+    assert flashcard.rating_flashcard == 1
+    assert flashcard.content_type_question == ContentType.objects.get_for_model(text.pk)
+    assert flashcard.object_id_question == 1
+    assert flashcard.content_type_answer == ContentType.objects.get_for_model(text.pk)
+    assert flashcard.object_id_answer == 1
+    assert isinstance(flashcard.date_modification, date)
+    assert flashcard.author == user.id
+    assert flashcard.difficulty == difficulty.pk
+
+
+def test_flashcard_fields(flashcard, remove_test_data):
+    assert [*vars(flashcard)] == [
+        "_state",
+        "id",
+        "deck",
+        "category",
+        "rating_flashcard",
+        "is_active",
+        "content_type_question",
+        "object_id_question",
+        "content_type_answer",
+        "object_id_answer",
+        "date_added",
+        "date_modification",
+        "author",
+        "difficulty",
+    ]

```

Comment:
I have and errors when I make docker compose exec backend pytest -k flashcard:
ERROR decks/tests/test_flashcard_model.py::test_correct_gui_representation - AttributeError: 'function' object has no attribute '_meta'
ERROR decks/tests/test_flashcard_model.py::test_flashcard_creation - AttributeError: 'function' object has no attribute '_meta'
ERROR decks/tests/test_flashcard_model.py::test_flashcard_fields - AttributeError: 'function' object has no attribute '_meta'

I don't know if you have also this mistake or these are only at my computer.

Reply:
changed and fixed pytest unit tests and fixture for flashcard model in conftest.py

Reply:
for me everything is good now